ethereum_history_api/circuits/lib/src/verifiers/header.nr

```diff
@@ -49,24 +49,21 @@ pub fn verify_header(
         expected_header_fields_count == header_rlp_list.len, "number of header RLP fields does not match"
     );
 
-    header_rlp_list.get(BLOCK_NUM_INDEX).assert_eq_u64(
-        "Block number",
-        block_header_rlp.data,
-        block_header_partial.number
-    );
+    let block_header_rlp = Fragment::new_with_length(block_header_rlp.length as u64, block_header_rlp.data);
```

Comment:
Please add a task to this sprint to convert block_header_rlp to BoundedVec. It will be a nice filler for someone

Reply:
Done